### ISSUE TITLE
[BUG] Phalcon can't handle models with AI PK as only column

### ISSUE THREAD

**Issue by scrnjakovic** [phalcon/cphalcon#13517]
### Expected and Actual Behavior

I want to have a model which serves as some sort of base reference, and it has only one column: auto incremented PK.

Trying to save such model results in exception being thrown by `Phalcon\Db\Adapter` [line 319](https://github.com/phalcon/cphalcon/blob/caeac66e7db02fd54c44a27cf22a6381654af2ee/phalcon/db/adapter.zep#L319)

* Phalcon version: 3.4.0
* PHP Version: PHP 7+
* Operating System: macOS
* Other related info (Database, table schema): MySQL 8.0.12

**Comment by nadorator** [comment]
I don't think you need a MySQL table with only an AI PK it is non sens for both relational schema and performance issues.  Perhaps you just need it in your php model to inherit from it? if yes then just do an abstract class or something similar. Could you be more precise with your data structures?

**Comment by scrnjakovic** [comment]
@nadorator I disagree. Sure I could do it differently but I'd then have to reinvent the wheel, auto increment values myself and enforce constraints manually. I'd rather have one extra table and have everything I need out-of-box than get rid of the table and reimplement those things myself.
So, in my case it makes perfect sense, but I don't think this should be up for debate as it's too broad.
